CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## 1.0.1 (Unreleased)
+## 1.1.0 (Unreleased)
+
+IMPROVEMENTS:
+
+* provider: Update Nomad to v0.7.0
+* provider: Allow setting ACL in hcl
+
```

Comment:
In this case it doesn't really matter, but we tend to update the CHANGELOG as a separate step, outside the PR process. Otherwise, merge conflicts quickly get out of control.